README.md

```diff
@@ -8,33 +8,37 @@
 
 ## Getting Involved
 
-We are engaging in a collaborative process to more clearly describe Data Together so that the initiative can be better understood by newcomers. This process began during an in-person meeting in Boston on August 29, 2017 with [@dcwalk](https://github.com/dcwalk), [@flyingzumwalt](https://github.com/flyingzumwalt), [@jeffreyliu](https://github.com/jeffreyliu), [@mhucka](https://github.com/mhucka), [@ebarry](https://github.com/ebarry), [@b5](https://github.com/b5), [@jschell42](https://github.com/jschell42), [@titaniumbones](https://github.com/titaniumbones), [@rgardaphe](https://github.com/rgardaphe). We are starting with time-tested patterns from non-profits, collaboratives, and companies: Mission, Vision, Values. Please click on the links below to see and comment on the draft text.
-
-If you would like to **add to the discussion and submit changes**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md). 
+If you would like to **add to the discussion and submit changes**, please see our [Contributing Guidelines](./CONTRIBUTING.md) and [Code of Conduct](https://github.com/datatogether/datatogether/blob/master/CONDUCT.md).
 
 If you want to **get into the code and start hacking**, then [webapp](https://github.com/datatogether/sentry) and [sentry](http://github.com/datatogether/sentry) are the best points of entry.
 
 We use GitHub issues for [tracking ideas and errors](https://github.com/datatogether/datatogether/issues) and Pull Requests (PRs) for [submitting changes](https://github.com/datatogether/datatogether/pulls). Every repository has its own issue tracker so please be sure to post your issues in the best place!
 
-## Mission
+Many of the people behind Data Together are active participants in the [Archivers Slack team](https://archivers-slack.herokuapp.com/), which anyone can join. We use the #datatogether channel.
 
-_Our purpose for existing, our charge, our goal, what this initiative does. (1 succinct sentence only)_
+## Mission (work in progress)
 
-Data Together helps communities steward the data they care about.
+We're building a better future for data.
 
-_[if there’s extra on “how” we do our work, add **[model](/model.md) / strategy / strategic plan / initiative areas**]_
+Data Together is a rallying point for everyone who believes that people should be able to possess copies of the data they rely on, and who see that communities should be using decentralized technologies in order to hold data as collective assets.
 
-## Vision 
+## Vision (work in progress)
 
 _Our future abilities and role in broader movement, NOT a utopian static state. Meant to be inspirational to ourselves and partners to strive together, and can be more flowery than the mission_.
```

Comment:
I now find these italics distracting (knowing full well this isn't part of your changes @flyingzumwalt)

Reply:
Yeah see my comment above about re-introducing the framing text above, or alternatively re-jigging for all sections.

Reply:
For now I will reinstate all of the italicized framing text (accidentally deleted some lines but left most). We can re-work that in a separate PR after we land this one (or reject it).